backend/.env.test

```diff
@@ -0,0 +1,14 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "src" folder
+FLASK_APP=src/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing albums in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Spotify Api
+SPOTIFY_CLIENT_ID=https://developer.spotify.com/dashboard
+SPOTIFY_SECRET=https://developer.spotify.com/dashboard
+SPOTIFY_REDIRECT_URI="http://localhost:8080/spotify-redirect"
```

Comment:
Can't imagine it will bite you here, but worth noting I default recommend that you don't put quotes around env variables in these files - Python's dotenv doesn't care, but docker interprets them as literally part of the string when we pass them through directly and it can lead to a very confusing debugging experience!